src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -12,313 +12,41 @@
 #ifndef MLPACK_METHODS_ANN_LAYER_LAYER_TYPES_HPP
 #define MLPACK_METHODS_ANN_LAYER_LAYER_TYPES_HPP
 
-#include <boost/variant.hpp>
+#include <mlpack/methods/ann/layer/layer.hpp>
+#include <mlpack/methods/ann/layer/multi_layer.hpp>
+#include <mlpack/methods/ann/layer/recurrent_layer.hpp>
```

Comment:
I am unable to find this file. Did it got removed in later commit?

Reply:
Nope, I just forgot to add it.  Sorry about that!  It's added now.